### ISSUE TITLE
Update code standards from ES5 to ES6/7

### ISSUE THREAD

**Issue by karmacoma** [LiskHQ/lisk-js#63]
We should update the code standards from ES5 to ES6/7. Initially using eslint which is already in-place. This should be done after #62 is resolved.

**Comment by alepop** [comment]
@karmacoma you mean refactor code to use `let`, `const`, `class`, `import` and other es6 standard feature?

**Comment by willclarktech** [comment]
@karmacoma Perhaps we should arrange a meeting to establish which standards to adopt?

**Comment by willclarktech** [comment]
Proposal: follow the guidelines in our (viz [airbnb's](https://github.com/airbnb/javascript/blob/master/README.md)) eslint config. E.g. 
- prefers `let` and `const` over `var`
- prefers `const` over `let` especially where there's no reassignment
- prefers property value shorthand
- etc...

**Comment by willclarktech** [comment]
Superseded by #192
